.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contribution Guidelines
+
+Want to contribute to this repository? Please read below first:
+
+- [Issues and Bugs](#issues-and-bugs)
+- [Feature Requests](#feature-requests)
+- [Doc Fixes](#doc-fixes)
+- [Submission Guidelines](#submission-guidelines)
+
+## Issues and Bugs
+
+If you find a bug in the source code or a mistake in the documentation, you can help us by
+submitting an issue to this repo. Even better you can submit a Pull Request with a fix.
+
+**Please see the Submission Guidelines below**.
+
+## Feature Requests
+
+You can request a new feature by submitting an issue to this repo. Proposed features (with suitable design documentation and reasoning) can be crafted and submitted to this repo as a Pull Request.
+
+**Please see the Submission Guidelines below**.
+
+## Doc Fixes
+
+If you want to help improve the docs, it's a good idea to let others know what you're working on to minimize duplication of effort. Comment on an issue to let others know what you're working on, or create a new issue if your work doesn't fit within the scope of any of the existing doc fix projects.
+
+**Please see the Submission Guidelines below**.
+
+## Submission Guidelines
+
+### Setup
+
+1. Fork the project by navigating to the main [repository](https://github.com/IBM/carbon-addons-dotcom) and clicking the **Fork** button on the top-right corner.
+
+2. Navigate to your forked repository and copy the **SSH url**. Clone your fork by running the following in your terminal:
+
+   ```
+   $ git clone git@github.com:{ YOUR_USERNAME }/carbon-addons-dotcom.git
+   $ cd carbon-addons-dotcom
+   ```
+
+   See [GitHub docs](https://help.github.com/articles/fork-a-repo/) for more details on forking a repository.
+
+3. Once cloned, you will see `origin` as your default remote, pointing to your personal forked repository. Add a remote named `upstream` pointing to the main `carbon-addons-dotcom`:
+
+   ```
+   $ git remote add upstream git@github.com:IBM/carbon-addons-dotcom.git
+   $ git remote -v
+   ```
+
+4. Switch to our version of Node. If you are using (nvm [Node Version Manager)](https://github.com/creationix/nvm), you can run `nvm use` to quicky switch Node versions. If you are aren't using nvm, you can still check the `.nvmrc` dotfile in the root of this project to find the major stable version of Node we are using, and then you can switch manually.
```

Comment:
Should the parenthesis be outside of the URL markup?

`(nvm [Node Version Manager)](...) --> (nvm [Node Version Manager](...)) `

Reply:
Based on this paragraph, it looks like we'll need to add a `.nvmrc` file.